docs/ml-classification-regression.md

```diff
@@ -374,6 +374,197 @@ regression model and extracting model summary statistics.
 
 </div>
 
+## Generalized linear regression
+
+When working with data that has a relatively small number of features (< 4096), Spark's GeneralizedLinearRegression interface
+allows for flexible specification of [generalized linear models](https://en.wikipedia.org/wiki/Generalized_linear_model) (GLMs) which can be used for various types of
+prediction problems including linear regression, Poisson regression, logistic regression, and others.
+
+Contrasted with linear regression where the output is assumed to follow a Gaussian
+distribution, GLMs are specifications of linear models where the response variable $Y_i$ may take on _any_
+distribution from the [exponential family of distributions](https://en.wikipedia.org/wiki/Exponential_family). 
```

Comment:
The familes and link function are documented in the table below. I could move the table, or are you suggesting something somewhat different?